index.html

```diff
@@ -1,13 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="index.css">
-    <title>Hello Microverse!</title>
+    <link rel="stylesheet" href="./index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <title>Document</title>
 </head>
-<body>
-    <h1>Hello Microverse!</h1>
-</body>
-</html>
\ No newline at end of file
+
+<c>
+<header>
+    <nav class="navbar">
+        <a href="/" class="logo">My Logo</a>
+        <img src="./assets/Menu.svg" alt="hamburger" class="hamburger"> 
+    </nav>
+</header>
+    <main class="main-container">
+        <section class="intro">
+          <div class="intro-text">
+                <p class="greeting h-text">I'm Alexandra <br> Glad to see <br> you!</p>
```

Comment:
- Hey, this text should your h1 as this is the main header of your page. let's fix this to improve your document layout.